<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>questWindow</class>
 <widget class="QMainWindow" name="questWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Corona questionnaire</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; text-decoration: underline;&quot;&gt;코로나19 감염증에 관한 문진표&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>561</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1. 최근 14일 이내에 코로나 관련 증상이 있습니까?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>    발열(37.5°C 이상)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>    임상증상(인후통, 근육통, 기침, 가래, 후각 손실, 미각 손실 등)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>    증상 없음</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>561</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2. 최근 14일 이내에 해외여행을 다녀온 적이 있습니까?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yradi_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>    예</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="nradi_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>    아니오</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="nradi_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>    아니오</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>561</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3. 최근 14일 이내에 확진자 또는 밀접접촉자와 접촉하신 적이 있습니까?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yradi_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>    예</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="yradi_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>620</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>    예</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>561</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>4. 최근 14일 이내에 코로나 검사(자가진단키트, PCR)를 한 적이 있습니까? </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="nradi_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>620</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>    아니오</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="submit_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>710</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>제출</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
